panic: invalid IP (length 0)

# 🐞 Bug Report

Beacon node crashes after a 
```console
panic: invalid IP(length 0)
```
I am on the Onyx testnet and the node was running for 5-10 hours prior to crashing.

### Has this worked before in a previous version?

I have only used the recent versions of the beacon-chain.  I have not had this problem before.

## 🔬 Minimal Reproduction

./prysm.sh beacon-chain --p2p-max-peers 999 --p2p-host-ip=$(curl -s v4.ident.me)

## 🔥 Error
<pre><code>
panic: invalid IP (length 0)

goroutine 67502238 [running]:
github.com/ethereum/go-ethereum/p2p/enode.nodeItemKey(0xaf27410898fd780c, 0x4235916104e8634a, 0xf6483e06ae73d196, 0x7011b7d9abe78b44, 0x0, 0x0, 0x0, 0x185f2e8, 0x8, 0xc026693980, ...)
    external/com_github_ethereum_go_ethereum/p2p/enode/nodedb.go:152 +0x491
github.com/ethereum/go-ethereum/p2p/enode.(*DB).FindFails(0xc00000f300, 0xaf27410898fd780c, 0x4235916104e8634a, 0xf6483e06ae73d196, 0x7011b7d9abe78b44, 0x0, 0x0, 0x0, 0x10000000164fbc0)
    external/com_github_ethereum_go_ethereum/p2p/enode/nodedb.go:384 +0x78
github.com/ethereum/go-ethereum/p2p/discover.(*lookup).query(0xc0076c4a20, 0xc01dc34fc0, 0xc028bed9e0)
    external/com_github_ethereum_go_ethereum/p2p/discover/lookup.go:144 +0xc3
created by github.com/ethereum/go-ethereum/p2p/discover.(*lookup).startQueries
    external/com_github_ethereum_go_ethereum/p2p/discover/lookup.go:127 +0x1be
</code></pre>


## 🌍  Your Environment

**Operating System:**

<pre>
  <code>
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04 LTS
Release:	20.04
Codename:	focal

Linux myVMWARE 5.4.0-37-generic #41-Ubuntu SMP Wed Jun 3 18:57:02 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

  </code>
</pre>

**What version of Prysm are you running? (Which release)**

<pre>
  <code>
Prysm/v1.0.0-alpha.11/87ca73d60512ef488fe9551237bc9601c0f34809. Built at: 2020-06-12 19:11:43+00:00
  </code>
</pre>

**Anything else relevant (validator index / public key)?**

--- a/deps.bzl
+++ b/deps.bzl
@@ -52,6 +52,7 @@ def prysm_deps():
         name = "com_github_prysmaticlabs_bazel_go_ethereum",
         build_file_generation = "off",
         importpath = "github.com/prysmaticlabs/bazel-go-ethereum",
+        replace = "github.com/ethereum/go-ethereum",
         sum = "h1:BgFL+G37WhgPOvkQveBXevApdusxtkrRoTzN9O3jnDM=",
         version = "v0.0.0-20200615030327-5f59060ced70",
     )

--- a/deps.bzl
+++ b/deps.bzl
@@ -52,16 +52,15 @@ def prysm_deps():
         name = "com_github_prysmaticlabs_bazel_go_ethereum",
         build_file_generation = "off",
         importpath = "github.com/prysmaticlabs/bazel-go-ethereum",
-        replace = "github.com/ethereum/go-ethereum",
-        sum = "h1:X44ghT3epjsrDWm1PRZ/aaQQcrl4y6/jtuge1sD32HY=",
-        version = "v0.0.0-20200421124922-0beb54b2147b",
+        sum = "h1:BgFL+G37WhgPOvkQveBXevApdusxtkrRoTzN9O3jnDM=",
+        version = "v0.0.0-20200615030327-5f59060ced70",
     )
 
     # Note: It is required to define com_github_ethereum_go_ethereum like this for some reason...
     # Note: The keep directives help gazelle leave this alone.
     go_repository(
         name = "com_github_ethereum_go_ethereum",
-        commit = "df74fa9e96217d0fffbe4f25788dd270b25243ed",  # keep
+        commit = "5f59060ced70d1107fd3df26bf248c95af388738",  # keep
         importpath = "github.com/ethereum/go-ethereum",  # keep
         # Note: go-ethereum is not bazel-friendly with regards to cgo. We have a
         # a fork that has resolved these issues by disabling HID/USB support and

--- a/go.mod
+++ b/go.mod
@@ -52,7 +52,6 @@ require (
 	github.com/libp2p/go-libp2p v0.9.2
 	github.com/libp2p/go-libp2p-blankhost v0.1.6
 	github.com/libp2p/go-libp2p-circuit v0.2.3
-	github.com/libp2p/go-libp2p-connmgr v0.2.3
 	github.com/libp2p/go-libp2p-core v0.5.6
 	github.com/libp2p/go-libp2p-crypto v0.1.0
 	github.com/libp2p/go-libp2p-host v0.1.0
@@ -72,6 +71,7 @@ require (
 	github.com/minio/sha256-simd v0.1.1
 	github.com/mohae/deepcopy v0.0.0-20170929034955-c48cc78d4826
 	github.com/multiformats/go-multiaddr v0.2.2
+	github.com/multiformats/go-multiaddr-net v0.1.5
 	github.com/olekukonko/tablewriter v0.0.4 // indirect
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/paulbellamy/ratecounter v0.2.0
@@ -118,6 +118,6 @@ require (
 	k8s.io/utils v0.0.0-20200520001619-278ece378a50 // indirect
 )
 
-replace github.com/ethereum/go-ethereum => github.com/prysmaticlabs/bazel-go-ethereum v0.0.0-20200530091827-df74fa9e9621
+replace github.com/ethereum/go-ethereum => github.com/prysmaticlabs/bazel-go-ethereum v0.0.0-20200615030327-5f59060ced70
 
 replace github.com/json-iterator/go => github.com/prestonvanloon/go v1.1.7-0.20190722034630-4f2e55fcf87b

